mayastor/src/bdev/nexus/nexus_rpc.rs

```diff
@@ -167,21 +167,29 @@ pub(crate) fn register_rpc_methods() {
         fut.boxed_local()
     });
 
-    jsonrpc_register("offline_child", |args: ChildNexusRequest| {
-        let fut = async move {
-            let nexus = nexus_lookup(&args.uuid)?;
-            nexus.offline_child(&args.uri).await
```

Comment:
why are we not returning the state now?

Reply:
When calling the offline/online rpc methods, the rpc clients were complaining that they were receiving a return value (the nexus state) when they weren't expecting to get anything back (they were expecting a unit type back which is () or empty). If you look at the child_operation rpc implementation in mayastor_svc.rs we were doing something similar i.e. ignoring the returned value and returning null.